packages/vue/src/pages/crm-contact-details/components/contact-form.vue

```diff
@@ -279,8 +282,10 @@ function startEdit() {
   isEditing.value = true;
 }
 
-function saveEdit() {
-  isEditing.value = false;
+function handleSaveClick({ validationGroup }: {validationGroup: Record<string, any>}) {
```

Comment:
In react and angular using ClickEvent and ButtonEvent exported from devextreme. Could it use there too?